### DISCUSSION TITLE
React Native support

### DISCUSSION THREAD

**Discussion by robertgaal** [phosphor-icons/react#17] [Ideas]
Hi folks! Thanks for creating this. We love your work.

Would you be open to port this to React Native? We're currently taking icons from your site, running them through [SVGR](https://react-svgr.com/playground/), and then adding them to our own components with [react-native-svg](https://github.com/react-native-svg/react-native-svg). 

This works, but a library like this one, but for React Native, would be easier to implement and update. [We](https://quest.audio) would be happy to contribute to sponsorship if needed!

**Comment by rektdeckard** [comment]
Belated, but closing due to existence of [duongdev/phosphor-react-native](https://github.com/duongdev/phosphor-react-native). We probably won't be writing a first-party port with this lib as the established solution.
